analysis/configuration/configs.py

```diff
@@ -7,6 +7,7 @@ class PercentChange:
     dimensions: list = attr.ib()
     sort_by: list = attr.ib()
     include_dimension_permutations: bool = attr.ib(True)
+    report_rounding: int = attr.ib(2)
```

Comment:
maybe call this something else since it isn't exclusive to report data format?

Reply:
Changed to `results_rounding`